{"schemaVersion":{"minor":3,"major":0,"patch":0},"abstract":[{"type":"text","text":"An unknown error occured while writing an image to a file."}],"metadata":{"externalID":"s:10ScanbotSDK24SBSDKImageProcessorErrorO05writeE0yA2CmF","role":"symbol","roleHeading":"Case","modules":[{"name":"ScanbotSDK"}],"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"writeError","kind":"identifier"}],"symbolKind":"case","title":"SBSDKImageProcessorError.writeError"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK","doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError"]]},"sections":[],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"writeError"}]}],"kind":"declarations"}],"identifier":{"url":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError\/writeError","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/scanbotsdk\/sbsdkimageprocessorerror\/writeerror"]}],"references":{"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK/SBSDKImageProcessorError":{"abstract":[{"text":"An enum of possible errors when processing images.","type":"text"}],"identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError","title":"SBSDKImageProcessorError","kind":"symbol","url":"\/documentation\/scanbotsdk\/sbsdkimageprocessorerror","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"SBSDKImageProcessorError","kind":"identifier"}],"type":"topic","navigatorTitle":[{"text":"SBSDKImageProcessorError","kind":"identifier"}]},"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK":{"kind":"symbol","identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK","abstract":[{"type":"text","text":"Version: 5.1.4"}],"url":"\/documentation\/scanbotsdk","type":"topic","role":"collection","title":"ScanbotSDK"},"doc://net.doo.ScanbotSDK/documentation/ScanbotSDK/SBSDKImageProcessorError/writeError":{"url":"\/documentation\/scanbotsdk\/sbsdkimageprocessorerror\/writeerror","abstract":[{"text":"An unknown error occured while writing an image to a file.","type":"text"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"writeError","kind":"identifier"}],"role":"symbol","type":"topic","title":"SBSDKImageProcessorError.writeError","identifier":"doc:\/\/net.doo.ScanbotSDK\/documentation\/ScanbotSDK\/SBSDKImageProcessorError\/writeError","kind":"symbol"}}}